cdap-common/src/main/java/co/cask/cdap/common/service/AbstractRetryableScheduledService.java

```diff
@@ -133,6 +133,16 @@ protected final void shutDown() throws Exception {
     }
   }
 
+  /**
+   * Called when the RetryStrategy used by this class returns a negative number, indicating to abort the operation.
+   *
+   * @param e the exception which exhausted retries
+   * @return the number of milliseconds to delay until the next call to this method
```

Comment:
This is not accurate. It is the number of ms to delay until the next iterate gets triggered again.